specs/Standards/Wallets/InjectedWallets.md

```diff
@@ -64,7 +64,7 @@ interface Network {
 }
 
 interface SignInParams {
-  permission: transactions.FunctionCallPermission;
+  permissions?: Array<transactions.FunctionCallPermission>;
```

Comment:
1. What is the expected behaviour from a wallet when this method is called without a `permissions` property?

2. If a caller submits a payload matching the existing specification, with the non-pluralized shape `{ permission, accounts }`, what do we expect a wallet that conforms to this updated specification do?

3. This will be a breaking change for all wallets that have already implemented the specification with `permission` as the property name rather than `permissions`.  If a caller implements the spec as you are proposing in this PR, but they submit it to a wallet that has not yet updated their interface to be aware of the new pluralized property name, what will that wallet do?

Reply:
1. The wallet will not create any function call access keys. Just like with the MyNearWallet `requestSignIn` method optionally takes in `contractId` as a parameter. It its not set no function call access keys will be created.
2. We can create a check for both specs for now.
3. I'm assuming that wallets are still in the progress of updating their interfaces to align with this NEP. The faster we update them about more changes it will better. This will also be a problem in for dApps after wallets implement the new interface. They will have to update their way of communicating with the wallet (unless they are using wallet-selector). It is recommended to use the wallet-selector for interacting with wallets so just updating wallet-selector interface will be enough. I'm not sure about this one though. Does this make sense?